docs/server-request.md

```diff
@@ -22,13 +22,36 @@ A low-level action that allows for any kind of HTTP request to be performed.
 - [Parse JSON](/docs/Actions/CallApi.md) - creates variables from JSON returned by API calls
 - [Regex](/docs/Actions/CallApi.md) - create variables by matching patterns inside the content
 
-## `Parameter Reference`
+## `Input Parameter Reference`
 
 | Parameter     | Description                           | Supports Variables |
 |---------------|---------------------------------------|--------------------|
-| URL           | Absolute URL where the resource to be invoked lives. | Yes |
-| HTTP Method   | The operations to perform against the URL.<br/> Possible values are GET, POST, PUT, DELETE, HEAD. | No |
+| URL           | The URL where the resource to be invoked lives. It can be relative or absolute. | Yes |
+| Enforce SSL   | This will for the request to be sent to a secure version of the URL(https instead of http). This is useful for the instances in which the URL comes from a variable and you want to ensure it is secured.| No |
+| Timeout       | The amount of time in seconds in which the response must come or the request fails. If nothing is provided, it will default to 100 seconds. If you don't want to wait for the request to finish(when you must leverage higher Timeout values) we suggest using [Execute Actions Async](/docs/Actions/ExecuteAsync.md). | Yes |
+| HTTP Method   | The operations to perform against the URL.<br/> Possible values are GET, POST, PUT, DELETE, HEAD, PATCH. | No |
 | Data          | If the HTTP Method is POST or PUT, data is allowed to be sent to the target URL. The data can take various forms, from key-value pairs to complex JSON. For the target URL to understand the format, it needs to be passed through the Content-Type header. | Yes |
+|Do not Escape Tokens in Data | By default, we escape tokens in order to not break the XML structure. If you need your tokens to bring XML data, you will have to check this. | No|
```

Comment:
Does this only apply to XML? How about JSON or application/form-data?
What does "XML data" mean? Is it a CDATA section?

Reply:
Let's avoid "check this" as it's too confusing. _Check these_ proposals below instead:

Enable this option when you need tokens to bring XML data.
Tick this option when you need tokens to bring XML data